Sources/StreamChat/Controllers/CurrentUserController/CurrentUserController.swift

```diff
@@ -252,6 +252,17 @@ public extension CurrentChatUserController {
             }
         }
     }
+
+    /// Fetches all the unread information from the current user.
```

Comment:
We could improve the docs here:
- for the current user
- add the completion parameter
- one-time request is more appropriate than ad-hoc I believe?

Reply:
We could improve the docs here:

> for the current user
✅ 

> add the completion parameter
I think this is kinda self explanatry, usually our completion docs are a bit redudant, WDYT?

> one-time request is more appropriate than ad-hoc I believe?
Yes I will improve this 👍